test/mqtt_ascoltatore_spec.js

```diff
@@ -1,3 +1,70 @@
+it("should re-subscribe to topics", function(done) {
```

Comment:
I can at it to describeAscoltatore but wasn't sure how it was called and it can't be run in parallel with anything else. It needs to be run once in series as to not cause port binding conflicts.